libraries/chain/include/eosio/chain/global_property_object.hpp

```diff
@@ -66,11 +66,25 @@ namespace eosio::chain {
 
          std::optional<block_num_type>       proposed_schedule_block_num;
          producer_authority_schedule         proposed_schedule;
-         chain_config                        configuration;
+         chain_config_v1                     configuration;
          chain_id_type                       chain_id;
          kv_database_config                  kv_configuration;
          wasm_config                         wasm_configuration;
       };
+      struct snapshot_global_property_object_v6 {
+         // minimum_version and maximum_version refer to chain_snapshot_header
+         // version. snapshot_global_property_object_v6 applies to version 7 & 8
+         static constexpr uint32_t minimum_version = 7;
+         static constexpr uint32_t maximum_version = 8;
+         static_assert(chain_snapshot_header::minimum_compatible_version <= maximum_version,
+                       "snapshot_global_property_object_v6 is no longer needed");
+
+         std::optional<block_num_type>       proposed_schedule_block_num;
+         producer_authority_schedule         proposed_schedule;
+         chain_config_v1                     configuration;
```

Comment:
`chain_config_v2`. How did tests pass? Need a test with non-default values.

Reply:
Nice, `kv_configuration` is gone.

Reply:
I think `chain_config_v1` is right here since this for v7 & v8 snapshots.

Reply:
Yes, this is for V7 and V8. That's why `chain_config_v1`.